### PR TITLE
improve setup of backend-server

### PR DESCRIPTION
**Pull request by Katherinf**
This is nodemon configuration.

### CODE REVIEW

File: src/backend/App.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 const express = require('express')
```
Comment (rmallols) [inline src/backend/App.js]: Missing semi-colons here :)

File: package.json [hunk 2]
```diff
@@ -13,7 +13,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "start-backend": "nodemon ./src/backend/App.js localhost 3001"
```
Comment (rmallols) [inline package.json]: I believe you don't really need `localhost 3001` here

**Review by rmallols — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (Katherinf) [reply package.json]: Ok, So I just need to delete and that set?

Comment (rmallols) [reply package.json]: Yep, the line should look like:

```
"start-backend": "nodemon ./src/backend/App.js"
```

**Review by Katherinf — COMMENTED** [review]
Fixed changes from pull request

File: src/backend/App.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-const express = require('express')
+const express = require('express');
 const app = express()
```
Comment (rmallols) [inline src/backend/App.js]: Still some semi-colon missing on this file :)

Comment (rmallols) [reply src/backend/App.js]: Lines number 2 and 3

**Review by rmallols — CHANGES_REQUESTED** [review]

**Pull request merged**